apps/extension/src/components/password-input.tsx

```diff
@@ -0,0 +1,45 @@
+import { EyeClosedIcon, EyeOpenIcon } from '@radix-ui/react-icons';
+import { useState } from 'react';
+import { Input, InputProps } from 'ui/components';
+
+interface IPasswordInputProps {
+  value: string;
+  label: string;
+  isHelper: boolean;
+  helper: string;
```

Comment:
Helper is the wrong name for this. These are more like validations. Why not make this general? Something like:

```typescript
interface PasswordInputProps {
   validations: Validation[]
}

interface Validation {
   checkFn: (txt: string) => boolean)
   type: "warn" | "error" // corresponds to red or yellow
}
```